doc/sphinx/tutorial/learn-what-mir-can-do.md

```diff
@@ -1,135 +1,129 @@
-# Learn What Mir Can Do
-This tutorial will guide you through using some example programs that showcase
-Mir's capabilities in different setups. You will see how Mir supports a range
-of capabilities that can be used by Mir based compositors. Mir makes it easy to
-write compositors as shown in the [developer
-tutorial](write-your-first-wayland-compositor).
+# Getting started with Mir
 
+This tutorial will guide you through Mir's basic functionality. 
 
-## Installation
+By the end of the tutorial, you will install and run a demo application, learn how to use Mir in different environments, and learn of Mir features that can be used by Mir-based compositors. 
+
+If you are unfamiliar with Mir, start with this tutorial, and then proceed to a [developer tutorial](write-your-first-wayland-compositor) which will guide you through the process of writing compositors.
+
+
+## Installing demo applications
 Mir demos are available on Debian derivatives, Fedora, and Alpine. For distros
-that don't have prebuilt binaries, examples can be built from source.
+that don't have pre-built binaries, examples can be built from source.
 
-To install Mir demos on Debian and its derivatives:
+### On Debian and its derivatives
 
 ```sh
 sudo apt install mir-demos mir-graphics-drivers-desktop
 ```
 
-<details>
-<summary> Installing Demos on Other Distros </summary>
-
-Installing Mir demos on Fedora
+### On Fedora
 ```sh
 sudo dnf install mir-demos
 ```
 
- Installing Mir demos on Alpine
+### On Alpine
 ```sh
 sudo apk add mir-demos mir
 ```
-</details>
 
+## Running applications on X11 or Wayland
 
-## Running
-The main script you'll want to play around with is `miral-app`. It runs a shell
-with some eyecandy by default, but you can run it in kiosk mode if that fits
-your usecase more.
+You will work with a `miral-app` script that runs a shell with basic GUI.
 
-For the uninitiated: The shell mode runs a floating window manager just like
-GNOME and such where you can move windows and maximize or minimize them. Kiosk
-mode on the other hand assumes you want one (or more) applications in
-fullscreen mode all the time.
+Mir allows you to run applications in the *shell mode* or in *kiosk mode*.
```

Comment:
It comes from the pre-existing text but "shell mode" is a bad term: all the modes are types of shell.

```suggestion
Mir allows you to run a graphical shell in *desktop mode* or in *kiosk mode*.
```